polaris/ocean/tests/baroclinic_channel/default/__init__.py

```diff
@@ -27,7 +28,10 @@ def __init__(self, test_group, resolution):
 
         self.add_step(
             Forward(test_case=self, ntasks=4, min_tasks=4, openmp_threads=1,
-                    resolution=resolution, run_time_steps=3))
+                    resolution=resolution, run_time_steps=None))
```

Comment:
Ah, no, not 10 minutes, 10 days!  I don't think we want that.

Reply:
My plan is just to set it back to `run_time_steps=3`. I only set it to None for testing so that the velocity field was a little stronger by end of simulation.